.github/ISSUE_TEMPLATE/bug_website.yml

```diff
@@ -0,0 +1,52 @@
+name: Website Bug Report
+description: This type of issue is for problems with the design or functionality of the website
+labels:
+  - "bug"
+  - "design"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thank you for reporting a bug! Please provide any information you think can help. Screenshots are a great help as well!
+        ### Also please check to see if someone else has already reported this bug!
+  - type: input
+    id: address
+    attributes:
+      label: URL
+      description: URL where the bug was found
+      placeholder: https://jupiterbroadcasting.com/
+  - type: dropdown
+    id: devices
+    attributes:
+      label: What devices did you see this bug on?
+      multiple: true
+      options:
+        - Phone
+        - Tablet
+        - Desktop
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
```

Comment:
Is there a way to do an other option for this? There are some quite unique [browser options out there](https://destinationlinux.org/episode-269/). Most of them are all based on these, but some of them aren't.  So leaving that option open would probably be a good idea (and maybe an optional box below to say what browser if they chose other?), because I imagine it'll be those browsers we'll see the most bugs with 😅

Reply:
I stole the dropdown idea from somewhere online. I just threw in the big 4 as a start. I am thinking I can add an `input` field for people to type in their browser if it is not listed.